As part of this release we had [1 commit](https://github.com/TestUser/FakeRepository/commits/2.4.2) which resulted in [1 issue](https://github.com/gep13/FakeRepository/issues?q=milestone%3A2.4.2?closed=1) being closed.

I am some awesome milestone description.

__Feature__

- [__#5__](http://example.com/5) Issue 5
